docusaurus/docs/explore/rpc.md

```diff
@@ -27,10 +41,26 @@ Using the `poktrolld` binary:
 poktrolld query block --type=height 0 --node https://shannon-testnet-grove-seed-rpc.poktroll.com
 ```
 
-### gRPC
+## Alpha TestNet
+
+### Alpha RPC Endpoints
 
-_TODO_TECHDEBT: Add a gRPC example_
+We have provided `gRPC`, `JSON-RPC` and `REST` endpoints, which are available here:
 
-### REST
+- **RPC**: `https://shannon-testnet-grove-seed-rpc.alpha.poktroll.com`
```

Comment:
These are not active currently, but soon will be - I've already added the changes. Need to apply/test.